docs/actions/spm.md

```diff
@@ -47,6 +47,7 @@ Key | Description | Default
   `build_path` | Specify build/cache directory [default: ./.build] | 
   `package_path` | Change working directory before any other operation | 
   `configuration` | Build with configuration (debug|release) [default: debug] | 
```

Comment:
The `|` is [breaking](https://deploy-preview-608--fastlane-docs-preview.netlify.com/actions/spm/#parameters) it (and was [breaking](https://docs.fastlane.tools/actions/spm/) it before too).

I'll fix it and add a check to catch it in the future automatically.